R/read_habitatdata.R

```diff
@@ -452,6 +452,18 @@ read_habitatmap <-
 #' \href{https://github.com/inbo/n2khab-preprocessing}{n2khab-preprocessing}
 #' repository.
 #'
+#' @param keep_aq_patches Logical; \code{TRUE} by default.
+#' The data source \code{habitatmap_terr} aims at delineating all
+#' polygons with at least one (semi-)terrestrial type (i.e. patch).
+#' For those polygons, it returns all known habitat types and RIBs as patches.
+#' Hence, in several cases polygons do occur with a combination of terrestrial
+#' and aquatic patches.
+#' Setting \code{keep_aq_patches = FALSE} is convenient for use cases where one
+#' only wants to look at the (semi-)terrestrial patches: this setting will
+#' discard all aquatic patches in 'mixed' aquatic/terrestrial polygons.
+#' The aquatic types are defined as those with hydrological class \code{"HC3"}
+#' (see \code{tag_2}) in the \code{\link{types}} data source.
```

Comment:
Am I right that habitatmap_terr (polygons and patches) has a field ('tag2') with the code 'HC3' to designate the aquatic types?
It's maybe redundant to state here that this argument affects only the version with the patches. The polygons remain unaffected.

Reply:
It is the `types` data source that has a `tag_2` field; see the output of `read_types()`. It's good to know that my explanation can cause confusion; I'll make it more clear.

I will add that polygons remain unaffected, thanks for the suggestion.